### DISCUSSION TITLE
Embedded video thumbnail sizes

### DISCUSSION THREAD

**Discussion by Over9000Hogs** [stoatchat/discussions#750] [Polls]
I've spent a lot of time scrolling up in the process of backing up links and I personally don't like thumbnails. Please vote below so we can all see what the majority of this platform wants. Thank you for reading.

**Comment by cyrneko** [comment]
Why not both?

Element for example has small image previews in the timeline by default, with a bigger option too

Why not add something similar but for embeds? 🤔

**Comment by Anvnamnet** [comment]
I guess I can add to this nay neary a 2 years later. Why not have the choice of size AND the ability to not have one at all. (like discord - Sorry to blaspheme.)
